ArduPlane/quadplane.h

```diff
@@ -154,6 +154,9 @@ class QuadPlane
 
     // return true if the user has set ENABLE
     bool enabled(void) const { return enable != 0; }
+
+    // get value for pilot_velocity_z_max_dn
```

Comment:
This comment really doesn't add much except clutter :-)

Reply:
agree, but following the pattern...all get functions seem to have a comment header....

Reply:
Make it useful or remove it IMO.

Reply:
removed